deploy/examples/object.yaml

```diff
@@ -109,6 +109,11 @@ spec:
       disabled: false
     readinessProbe:
       disabled: false
+  # hosting:
+  #   The list of subdomain names for virtual hosting of buckets.
+  #   dnsNames:
+  #     - "my-store.rook-ceph.svc.cluster.local"
```

Comment:
Default service can't provide wildcards, so don't confuse users by including it.